db/schema.rb

```diff
@@ -73,15 +73,10 @@
   create_table "users", force: :cascade do |t|
     t.string   "name"
     t.string   "email"
-    t.datetime "created_at",          null: false
-    t.datetime "updated_at",          null: false
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
     t.string   "password_digest"
     t.string   "remember_digest"
-    t.string   "avatar_file_name"
```

Comment:
you need lines 80 through 84 through. just run a rails db:reset and it should take care of this